### ISSUE TITLE
Shadowy sprinty mob

### ISSUE THREAD

**Issue by woothie** [HeavensGate/Eternal#21]
Named "Faithless" possibly. An hallucination that sprints at you and dissipates into smoke/nothing as it reaches you (or hits a five second max timer to avoid being stuck). On impact: "[spookmob] /swings at [urist] and vanishes the moment before impact!". On attacking spookmob: "[urist] /swings at the [spookmob] with the [wepan], but hits nothing but smoke as [spookmob] dissipates" (or something of the like, discuss with lore). 

Note: Not actual hallucination in the traditional sense, because everyone must be able to see it. This for increased wtf-factor and possibly a moment of actual panic or confusion, if maybe a creeping feel of wtf as people around tell player they saw it too. Note of note: player might suspect admin intervention, contemplate on this somehow?

**Comment by Jacquar** [comment]
A little too direct, using faithless to repetitively rush people? Unsure. Stuff this direct, rapid, intense should be under direct admin control, I feel - for which macros/hotkeys for admin verbs would be useful. If we could have a single key bound to Invis Incorp Spawn Effect Smoke, for example, to disappear at will...

I am more inclined to create a wider variety of simple living mobs. I don't want to trust AI to overt spookin'.

**Comment by Jacquar** [comment]
Non-attacking, following shadow-thing. Future possibilities: Duplicates silhoette of subject, as though it is their own shadow come to life. More varied behaviours, possibility of becoming something more solid or hostile. Possibility of player control. 

Rollout: Admin-triggered, then random event generation of basic shadowthings, later possible integration with roundtypes, cultmagic, and finally dynamic integration in response to player behaviour - i.e. murders/haunting.

Rationale: A shadowy thing which attacks harmlessly isn't scary after the first time, it's just predictable and dull. A thing which sits inactive but watches you generates a lasting tension and uncertainty.

Fluff: Issue posters, memos, other materials warning people not to interact with shadowthings, confirming they're harmless (hinting they're not), and telling people to get on with their work.
